### PR TITLE
Remove UglifyJS2 issue workaround

### PR DESCRIPTION
**Pull request by Turbo87**
This has been fixed in `v2.4.17` (see https://github.com/mishoo/UglifyJS2/pull/566 and https://github.com/mishoo/UglifyJS2/commit/ad18689d926d25c7a25b95c630c2ad05b7b5f5b5) and we are already depending on `^2.7.0` so we don't seem to need this workaround anymore.

Removing those lines also seems to fix the issue described in https://github.com/ember-cli/ember-cli/issues/6781

I'm currently testing this change in production with Sentry attached.

/cc @rwjblue @ef4 @stefanpenner @MiguelMadero

### CODE REVIEW

**Comment by rwjblue** [comment]
Seems good to me, please report back once you have finished testing...

**Comment by Turbo87** [comment]
@rwjblue we have deployed this into production a few minutes ago and the first error that just came in via Sentry had a perfect stack trace without any source map decoding issues like in the screenshot in https://github.com/ember-cli/ember-cli/issues/6781

tl;dr this patch indeed seems to fix https://github.com/ember-cli/ember-cli/issues/6781

**Comment by MiguelMadero** [comment]
I will test this afternoon as well. Thanks for tagging me and looking into
this.

**Comment by Turbo87** [comment]
bildschirmfoto 2017-03-24 um 23 44 41

working very well on sentry now :)
